docs/features/models/NER.ipynb

```diff
@@ -0,0 +1,671 @@
+    "2. [Get started with the model](#2.-Get-started-with-the-model)\n",
+    "\n",
+    "3. [Use the model for prediction](#3.-Use-the-model-for-prediction)\n",
+    "\n",
+    "    3.1. [Predict using Python](#3.1-Predict-using-Python)\n",
+    "    \n",
+    "    3.2. [Predict using CLI](#3.2-Predict-using-CLI)\n",
+    "     \n",
+    "4. [Train the model on your data](#4.-Train-the-model-on-your-data)\n",
+    "    \n",
+    "    4.1. [Train your model from Python](#4.1-Train-your-model-from-Python)\n",
+    "    \n",
+    "    4.2. [Train your model from CLI](#4.2-Train-your-model-from-CLI)\n",
+    "    \n",
+    "5. [Evaluate](#5.-Evaluate)\n",
+    "    \n",
+    "    5.1. [Evaluate from Python](#5.1-Evaluate-from-Python)\n",
+    "    \n",
+    "    5.2. [Evaluate from CLI](#5.2-Evaluate-from-CLI)\n",
+    "    \n",
+    "6. [Models list](#7.-Models-list)\n",
+    "\n",
+    "7. [NER-tags list](#7.-NER-tags-list)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Introduction to the task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Named Entity Recognition (NER)** is a task of assigning a tag (from a predefined set of tags) to each token in a given sequence. In other words, NER-task consists of identifying named entities in the text and classifying them into types (e.g. person name, organization, location etc). \n",
+    "\n",
+    "**BIO encoding schema** is usually used in NER task. It uses 3 tags: B for the beginning of the entity, I for the inside of the entity, and O for non-entity tokens. The second part of the tag stands for the entity type.\n",
+    "\n",
+    "Here is an example of a tagged sequence:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "| Elon | Musk | founded | Tesla| in | 2003 | . |\n",
+    "| --- | --- | --- | --- | --- | --- | --- |\n",
+    "| B-PER | I-PER | O | B-ORG | O | B-DATE | O |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we can see three extracted named entities: *Elon Musk* (which is a person's name), *Tesla* (which is a name of an organization) and *2003* (which is a date). To see more examples try out our [Demo](https://demo.deeppavlov.ai/#/en/ner)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The list of possible types of NER entities may vary depending on your dataset domain. The list of tags used in DeepPavlov's models can be found in the [table](#7.-NER-tags-list)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Get started with the model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ygOmNC9R9l_y"
+   },
+   "source": [
+    "First make sure you have the DeepPavlov Library installed.\n",
+    "[More info about the first installation](http://docs.deeppavlov.ai/en/master/intro/installation.html)"
```

Comment:
"[More info about the first installation.](http://docs.deeppavlov.ai/en/master/intro/installation.html)"